Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Avoid trying to copy not included jquery.js file.

diff -Naurp twitter-bootstrap.orig/Makefile twitter-bootstrap/Makefile
--- twitter-bootstrap.orig/Makefile	2012-03-19 13:05:54.790355045 +0100
+++ twitter-bootstrap/Makefile	2012-03-19 13:20:09.314593347 +0100
@@ -18,7 +18,6 @@ docs: bootstrap
 	node docs/build
 	cp img/* docs/assets/img/
 	cp js/*.js docs/assets/js/
-	cp js/tests/vendor/jquery.js docs/assets/js/
 
 #
 # BUILD SIMPLE BOOTSTRAP DIRECTORY
